tests/foreman/performance/test_standard_prep.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/python
```

Comment:
I believe you can just drop these shebangs. A shebang is only appropriate when a module will be executed directly. That is not the case, as far as I can tell. This module will only be imported and used by some other bit of code.